requirements.txt

```diff
@@ -16,7 +16,7 @@ pandas >= 1.0
 numpy >= 1.17
 matplotlib >= 3.0.3
 pyarrow >= 0.16
-responses >= 0.12.0
+responses >= 0.12.0, <= 0.18.0
```

Comment:
I think we may also need to change setup.cfg

Reply:
Commits squashed and contraints of `responses` included in `setup.cfg`.

All GitHub actions checks pass.